Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.12-rc-alpine as base
```

```suggestion
ARG PYTHON_VERSION=3.12

FROM python:${PYTHON_VERSION}-rc-alpine as base
```

Due to the python version being referenced in multiple places at build time, it should be an `ARG`. `${PYTHON_VERSION}` can also be used further down